test/cmd/projects.sh

```diff
@@ -15,13 +15,18 @@ os::cmd::expect_failure_and_text 'oc projects test_arg' 'no arguments'
 os::cmd::expect_success 'oc login -u test -p test'
 os::cmd::expect_success_and_text 'oc projects' 'You are not a member of any projects'
 # add a project and expect text for a single project
-os::cmd::expect_success_and_text 'oc new-project test4; sleep 2; oc projects' 'You have one project on this server: "test4".'
-os::cmd::expect_success_and_text 'oc new-project test5; sleep 2; oc projects' 'You have access'
+os::cmd::expect_success_and_text 'oc new-project test4' 'Now using project "test4" on server '
+# avoid flakes in case of slow server update
+os::cmd::try_until_text 'oc projects' 'You have one project on this server: "test4".'
+os::cmd::expect_success_and_text 'oc new-project test5' 'Now using project "test5" on server '
+os::cmd::try_until_text 'oc projects' 'You have access to the following projects and can switch between them with '
+os::cmd::expect_success_and_text 'oc projects' 'test4'
+os::cmd::expect_success_and_text 'oc projects' 'test5'
 # test --skip-config-write
 os::cmd::expect_success_and_text 'oc new-project test6 --skip-config-write' 'To switch to this project and start adding applications, use'
-os::cmd::expect_failure 'oc config view | grep "namespace: test6"'
-os::cmd::expect_success 'sleep 2; oc projects | grep test6'
+os::cmd::expect_success_and_not_text 'oc config view -o jsonpath="{.contexts[*].context.namespace}"' '\btest6\b'
```

Comment:
`{.contexts[*].context.namespace}` gives you all the namespaces with a space in between them.  I am making sure that we don't match `test5 abctest6` while still matching `test5 test6`.